src/cnd/cnd.ts

```diff
@@ -69,6 +69,24 @@ export interface EthereumCallContractPayload {
   network: string;
 }
 
+export interface LndSendPaymentPayload {
```

Comment:
Hum, I would actually question whether `network` is indeed in the others? If you think we should add it here I think a new issue would be better to avoid scope creep.